astropy/cosmology/realizations.py

```diff
@@ -10,6 +10,7 @@
 from astropy.utils.decorators import deprecated
 from astropy.utils.state import ScienceState
 
+from . import _io  # noqa: F401
```

Comment:
Ruff rule [F401](https://beta.ruff.rs/docs/rules/unused-import/) is about unused imports. It's worth adding a comment here why `_io` is being imported if it is unused. The reason is not obvious from the code.

Reply:
Done!